docs/source/guide/explanation/algorithms/segmentation/semantic_segmentation.rst

```diff
@@ -55,31 +48,32 @@ Models
-+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------+---------------------+-----------------+
-| `Custom_Semantic_Segmentation_SegNext_S <https://github.com/openvinotoolkit/training_extensions/blob/develop/src/otx/recipe/semantic_segmentation/segnext_s.yaml>`_                  | SegNext-s              | 15.35               | 13.9            |
-+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------+---------------------+-----------------+
-| `Custom_Semantic_Segmentation_SegNext_B <https://github.com/openvinotoolkit/training_extensions/blob/develop/src/otx/recipe/semantic_segmentation/segnext_b.yaml>`_                  | SegNext-b              |   32.08             | 27.56           |
-+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------+---------------------+-----------------+
-
-All of these models are members of the same `Lite-HRNet <https://arxiv.org/abs/2104.06403>`_ backbones family. They differ in the trade-off between accuracy and inference/training speed. ``Lite-HRNet-x-mod3`` is the recipe with heavy-size architecture for accurate predictions but it requires long training.
-Whereas the ``Lite-HRNet-s-mod2`` is the lightweight architecture for fast inference and training. It is the best choice for the scenario of a limited amount of data. The ``Lite-HRNet-18-mod2`` model is the middle-sized architecture for the balance between fast inference and training time.
-
-Use `SegNext <https://arxiv.org/abs/2209.08575>`_ model which can achieve superior perfomance while preserving fast inference and fast training.
-
-In the table below the `Dice score <https://en.wikipedia.org/wiki/S%C3%B8rensen%E2%80%93Dice_coefficient>`_ on some academic datasets using our :ref:`supervised pipeline <semantic_segmentation_supervised_pipeline>` is presented. We use 512x512 image crop resolution, for other hyperparameters, please, refer to the related recipe. We trained each model with single Nvidia GeForce RTX3090.
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------+-----------------+-----------------+-----------------+
+| Recipe Path                                                                                                                                                                          | Complexity (GFLOPs) | Model size (M)  | FPS (GPU)       | iter time (sec) |
```

Comment:
How is GPU FPS measured?

Reply:
![image](https://github.com/openvinotoolkit/training_extensions/assets/68103869/15a48411-c42d-4684-8ea4-2ad0f26693d0)

Reply:
Is `self.model = torch.compile(self.model)` applied at some point?

Reply:
I used this snippet of code inside Engine just before training (trainer.fit()). So, if we use torch.compile, it should be applied

Reply:
As Vlad said, `torch.compile` exists in `OTXModel.setup()` but it's executed only if `self.torch_compile` is `True`. It seems that default value of it is set as `False` and any default recipe doesn't change that value, so `torch.compile` isn't executed as default.